systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicScalabilityUtils.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.utils.kafkaUtils;
+
+import io.strimzi.api.kafka.model.KafkaTopicSpec;
+import io.strimzi.systemtest.resources.crd.KafkaTopicResource;
+import io.strimzi.systemtest.templates.crd.KafkaTopicTemplates;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import io.strimzi.systemtest.resources.ResourceManager;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import io.strimzi.systemtest.enums.CustomResourceStatus;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import static io.strimzi.systemtest.Constants.INFRA_NAMESPACE;
+
+
+@SuppressWarnings("MismatchedQueryAndUpdateOfCollection")
+public class KafkaTopicScalabilityUtils {
+    private static final Logger LOGGER = LogManager.getLogger(KafkaTopicUtils.class);
+    protected static ResourceManager resourceManager = ResourceManager.getInstance();
+
+    public static void createTopicsViaK8s(ExtensionContext extensionContext, String clusterName, String topicPrefix,
+                                          int numberOfTopics, int numberOfPartitions, int numberOfReplicas, int minInSyncReplicas) {
+        LOGGER.info("Creating topics via Kubernetes");
+
+        for (int i = 0; i < numberOfTopics; i++) {
+            String currentTopicName = topicPrefix + i;
+            LOGGER.debug("Creating topic {}", currentTopicName);
+
+            CompletableFuture.runAsync(() ->
```

Comment:
Why do you need to execute`CompletableFuture.runAsync(...`??  If you execute just 
```java
resourceManager.createResource(extensionContext, false, KafkaTopicTemplates.topic(
                            clusterName, currentTopicName, numberOfPartitions, numberOfReplicas, minInSyncReplicas, INFRA_NAMESPACE).build()
```
this creates you a KafkaTopic without waiting (i.e., pseudo async). Using also `runAsync(...)` in this way is not correct. Why did you use it like this? :)

Reply:
I thought this was slightly faster than the regular approach, if it's clearly misuse i apologize, I'm new to async usage of CompletableFuture, i suppose what's wrong and caught your eye, was not completing a completable. If there is no difference between regular creation of resource, I'll remove it. Thank you for your comment.